upgrade-notes/storefront_20260113_100000.md

```
@@ -0,0 +1,93 @@
+
+##### Package Updates
+
+Major dependency updates that may affect your code:
+
+- `next`: 14.1.4 → 15.4.10
+- `@sentry/nextjs`: 7.99.0 → 9.38.0
+- `vitest`: 0.34.6 → 2.1.9
+
+If you have custom Sentry configuration or vitest tests, review compatibility with the new versions.
+
+#project-base-diff
```

markdownlint-cli2 (0.18.1)

No space after hash on atx style heading

(MD018, no-missing-space-atx)